src/Angor/Shared/Services/NetworkService.cs

```diff
@@ -123,6 +129,37 @@ public async Task CheckServices(bool force = false)
                 }
             }
 
+            foreach (var explorerUrl in settings.Explorers)
+            {
+                if (force || (DateTime.UtcNow - explorerUrl.LastCheck).Minutes > 10)
+                {
+                    explorerUrl.LastCheck = DateTime.UtcNow;
+
+                    try
+                    {
+                        var uri = new Uri(explorerUrl.Url);
+                        
+                         var heartbeatUrl = new Uri(uri, "api/stats/heartbeat").ToString();
```

Comment:
is this true for both mempool and blockcore explorers?

Reply:
for blockcore it's ok and for mempool I will add that endpoint